service-front/module/Application/src/Helpers/DTO/FormProcessorResponseDto.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers\DTO;
+
+use Laminas\Form\FormInterface;
+
+class FormProcessorResponseDto
+{
+    public function __construct(
+        private string $uuid,
+        private FormInterface $form,
+        private array $responseData,
+        private string $template,
+        private array $variables = [],
+    ) {
+    }
+
+    public function getUuid(): string
+    {
+        return $this->uuid;
+    }
+
+    public function getForm(): FormInterface
+    {
+        return $this->form;
+    }
+
+    public function getResponseData(): array
+    {
+        return $this->responseData;
+    }
```

Comment:
I can't work out what this is for, and can't see it being used anywhere. Perhaps removable now?